### PR TITLE
Mobile App: feat show list file channel mobile

### PR DESCRIPTION
**Pull request by hoangtranlehuy**
Mobile App: feat show list file channel mobile
https://github.com/nccasia/mezon/issues/5321

### CODE REVIEW

File: apps/mobile/src/app/components/ChannelFiles/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	};
+
+	return (
+		<ScrollView
+			style={{ height: Metrics.screenHeight / (Platform.OS === 'ios' ? 1.4 : 1.3) }}
+			contentContainerStyle={{ paddingBottom: size.s_50 }}
+			showsVerticalScrollIndicator={false}
+		>
+			<View style={styles.container}>
+				<FlatList data={allAttachments} renderItem={renderItem} />
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ChannelFiles/index.tsx]: change to fastlist

File: apps/mobile/src/app/components/ChannelFiles/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { style } from './styles';
+
+const ChannelFiles = memo(({ currentChannelId }: { currentChannelId: string }) => {
+	const { themeValue } = useTheme();
+	const styles = style(themeValue);
+	const allAttachments = useSelector(selectAllListDocumentByChannel(currentChannelId));
+
+	const renderItem = ({ item }: { item: AttachmentEntity }) => {
+		return <ChannelFileItem file={item} />;
+	};
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ChannelFiles/index.tsx]: miss key

File: apps/mobile/src/app/components/ChannelFiles/ChannelFileItem/index.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+const ChannelFileItem = memo(({ file }: ChannelFileItemProps) => {
+	const { themeValue } = useTheme();
+	const styles = style(themeValue);
+	const userSendAttachment = useAppSelector((state) => selectMemberClanByUserId2(state, file?.uploader ?? ''));
+	const userName = userSendAttachment?.user?.username;
+	const attachmentSendTime = convertTimeString(file?.create_time as string);
+
+	const onPressItem = () => {
+		openUrl(file?.url);
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ChannelFiles/ChannelFileItem/index.tsx]: use Linking.openURL(url)

**Review by phuocnguyenncc — APPROVED** [review]

**Comment by hoangtranlehuy** [comment]
Screenshot_2024-12-12-10-57-04-020_com mobile mezon

**Pull request merged**